pulsar-broker/src/main/java/org/apache/pulsar/broker/service/Consumer.java

```diff
@@ -65,7 +65,7 @@
 public class Consumer {
     private final Subscription subscription;
     private final SubType subType;
-    private final ServerCnx cnx;
+    private final PulsarServerCnx cnx;
```

Comment:
what is the motivation of renaming `ServerCnx` to `PulsarServerCntx`? Can we avoid renaming this file?

Reply:
Ultimately `Consumer` shall depend on the interface and not the implementation. It's just that I haven't done it yet. Do you want me to do it in this PR ? It's the next step of my work anyway.

Reply:
I'll do the change for `Consumer`. It will also make `Topic`, `NonPersistentTopic`, and `PersistentTopic` depend of `ServerCnx` instead of `PulsarServerCnx` so that's better.

Reply:
@sijie I've done the change for Consumer.